test/integration/spa.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <meta charset='utf-8'>
+  <title>SDK SPA Test</title>
+  <script src='sdk.js'></script>
+  <script>
+    const clientPort = 3000;
+    const redirectPort = 4000;
+    const clientUri = `http://localhost:${clientPort}`;
+    const redirectUri = `http://localhost:${redirectPort}/redirect?app_origin=${clientUri}`
+
+    // this file tests a redirect scheme with a separate front end client & back
+    // end server. This is the same scheme that will be used if a developer
+    // chooses to host their redirect on Smartcar's CDN. For the purposes of
+    // this test we hit a local redirect server where we provide the
+    // `app_origin` param ensuring it will post back to that `app_origin` param
+    const options = {
+      // required
+      clientId: 'clientId',
+      // checked against event origin for security
+      // Note: this test does not actually use any Smartcar instance methods for
+      // triggering OAuth. Therefore the only important part of this redirectUri
+      // is the host (`app_origin` value does not matter). This is because we
+      // manually load the redirect page with the correct `app_origin` query
+      // param further below
```

Comment:
nit: thoughts on removing this comment entirely? The redirectUri has the correct info now and I think just adds more complexity. I'd wanna match the real flow as much as possible